pipelines/zap/pipeline.yml

```diff
@@ -96,7 +96,7 @@ jobs:
       params:
         text: |
           :x: FAILED to scan properties.
-          <https://ci-tooling.cloud.gov/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME|View build details>
```

Comment:
@afeld btw, there is a new env var you can use here: ATC_EXTERNAL_URL for the URL bit, so you don't have to hard code into your pipeline any longer.. something like:

```<$ATC_EXTERNAL_URL/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME|View build details>```